doc/tutorials/dune-package-management/management.md

```diff
@@ -0,0 +1,150 @@
+# Managing Packages
+
+This section will introduce some more features of the package management.
```

Comment:
I've removed the whole file, because it felt like a grab-bag of random things, so I moved these things into the place in the tutorial where they fit the story better.